### PR TITLE
project: add child project references

### PR DESCRIPTION
**Pull request by konorlevich**

### CODE REVIEW

**Comment by cvbarros** [comment]
Hello @konorlevich 👋 ! This looks great, would be very handy to have those fields exposed.
If you don't mind please, would it be possible to add some test coverage for these new feature?

Once that's covered, this will be good to merge!
Thanks

**Comment by konorlevich** [comment]
Hello, @cvbarros!
It's enough?

File: teamcity/project_test.go [hunk 1]
```diff
@@ -199,6 +199,31 @@ func TestProject_GetRootByName(t *testing.T) {
...
+
+	actual, err = sut.GetByName(testProjectId)
+	require.NoError(t, err)
+	cleanUpProject(t, client, testProjectId)
+
+	assert.Equal(2, actual.ChildProjects.Count)
+	childProjects := make(map[string]string, 2)
+	for _, i := range actual.ChildProjects.Items {
+		childProjects[i.ID] = i.Name
+	}
```
Comment (cvbarros) [inline teamcity/project_test.go:222]: What would be this `Name` collection for? Did you intend on asserting those names?
Either way the ID assertion is enough, up to you if you want to:
1- Assert the names
or
2- Remove the block

Comment (konorlevich) [reply teamcity/project_test.go:222]: Thanks, I forgot to add assertions

**Comment by cvbarros** [comment]
Thanks a lot @konorlevich 🙇 ! 🎉

**Pull request merged**